### PR TITLE
Fixed couple of other issues 

### PR DESCRIPTION
**Pull request by salehparsa**
I fixed couple of other issues and typo, and we can run the script properly. Haven't tested with latest JIRA yet

### CODE REVIEW

File: script.rb [hunk 1]
```diff
@@ -36,34 +36,34 @@ def initialize(config = Config.new)
...
-    new_field = config.new_custom_url
+    old_field_value = issue["fields"][config.old_custom_field]
+    new_field = config.new_custome_field
 
     req = Net::HTTP::Put.new post_uri.request_uri
     req.basic_auth(config.username, config.password)
     req["Content-Type"] = "application/json"
     req.body = "{\"fields\":{\"#{new_field}\":\"#{old_field_value}\"}}"
     Net::HTTP.start(post_uri(issue["key"]).host, post_uri(issue["key"]).port, :use_ssl => config.project_uri.scheme == "https", :verify_mode => OpenSSL::SSL::VERIFY_NONE) do |http|
+      response = http.request req
```
Comment (hosseintoussi) [inline script.rb]: i dont see where you use this `response`. i would vote for removing if not used.

**Review by hosseintoussi — CHANGES_REQUESTED** [review]
thanks man, good job 👏 🥇 

Just one minor point in the comment.
Also rebase this off of master and fix the conflicts.

**Comment by salehparsa** [comment]
Fixed the conflict as well @hosseintoussi

File: script.rb [hunk 2]
```diff
@@ -58,12 +57,16 @@ def update_issue(issue)
   end
 
   def post_uri(key)
-    URI("#{config.base_url}/rest/api/2/issue/#{key}".strip)
+    URI("#{base_url}/rest/api/2/issue/#{key}".strip)
   end
 
   def all_issues
     @_all_issues ||= Net::HTTP.start(config.project_uri.host, config.project_uri.port, :use_ssl => config.project_uri.scheme == 'https', :verify_mode => OpenSSL::SSL::VERIFY_NONE) do |http|
+<<<<<<< HEAD
```
Comment (hosseintoussi) [inline script.rb]: oops.. looks like there are some leftovers from the conflicts.

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1 @@
+*.yml
```
Comment (hosseintoussi) [inline .gitignore:1]: if you are ignoring this file, then it would be nice to rename `config.yml` file to something like `config.yml.example`

**Review by hosseintoussi — CHANGES_REQUESTED** [review]
Looks good 💯 . Just one more minor thing

File: .gitignore (hunk 3 above)
Comment (salehparsa) [reply .gitignore:1]: done sir 👍

**Comment by hosseintoussi** [comment]
merging it, thanks 👏  👏  👏  👏

**Pull request merged**